modules/serverless-rn-known-issues.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// serverless/release-notes.adoc
+
+[id="serverless-rn-known-issues_{context}"]
+= Known issues
+
+////
+Consequence - What user action or situation would make this problem appear? What did the customer experience as a result of the issue? What was the symptom?
+Cause (if it has been identified) - Why did this happen?
+Workaround (If there is one)- What can you do to avoid or negate the effects of this issue in the meantime?  Sometimes if there is no workaround it is worthwhile telling readers to contact support for advice. Never promise future fixes.
+Result - If the workaround does not completely address the problem.
+////
+
+These limitations exist in {ProductName} at this time:
```

Comment:
s/These limitations exist/The following limitations exist/

Does it make sense to say _in [product version]_? instead of _at this time_?